ocfweb/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from ocfweb.api import api
+
+urlpatterns = [
+    url(r'sign\.json', api.sign_text, name='sign-json'),
```

Comment:
what about calling it `/api/hours`? there's nothing in here really specific to the sign, and we might find other uses for it